Setup the React Components project (#60)
ChainSafe/web3-circle-libs: packages/circle-react-components/README.md

# ⚛️⚡ Web3 Circle React Components

A project containing the Web3 Circle React Components.

---

> Note: You are welcome to contribute. And if you want to create an MR check the file ./MAINTAINERS_NOTES.md.

Reviewer: ```suggestion
Note: You are welcome to contribute. And if you want to create a Pull Request check the file ./MAINTAINERS_NOTES.md.
```
